.changeset/two-buttons-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+'blocks-ui': minor
```

Comment:
```suggestion
'blocks-ui': patch
```

Reply:
thanks for all the suggested changes! nice and easy to commit